### DISCUSSION TITLE
Using kebab-case for css properties in objects is not supported. Did you mean &:hover, &[dataHover]?

### DISCUSSION THREAD

**Discussion by ghost** [chakra-ui/chakra-ui#5686] [Q&A]
The browser console showing this error message and i found this is not the only css properties getting this error from library. I have no idea which part goes wrong. Anyone facing this error?
my code

```
 <Link
            color={mainText}
            href="#"
            bg="inherit"
            borderRadius="inherit"
            fontWeight="bold"
            _hover={{ color: { mainText } }}
            _active={{
              bg: "inherit",
              transform: "none",
              borderColor: "transparent",
            }}
            _focus={{
              boxShadow: "none",
            }}
          >
            {brandText}
          </Link>
```
 package.json
```
    "@chakra-ui/icons": "^1.0.14",
    "@chakra-ui/react": "^1.8.6",
    "@chakra-ui/theme-tools": "^1.3.6",
    "@emotion/cache": "^11.4.0",
    "@emotion/react": "^11.7.1",
    "@emotion/styled": "^11.6.0",
```
![image](https://user-images.githubusercontent.com/37526525/156717996-334d10e2-4a10-4ece-b10b-76ba999353a6.png)
![image](https://user-images.githubusercontent.com/37526525/156718061-8d1a6931-75c9-4cbf-9d6b-fb1a2e1b913b.png)
![image](https://user-images.githubusercontent.com/37526525/156724467-ed4e0018-134c-431a-aaf1-2e24cdb00c7c.png)

**Comment by nelsonomuto** [comment]
yes; facing that error as well when the style was input as such

```
<Input  {...inputProps}  _focus={{ md: isInputFocused ? { boxShadow: 'outline' } : 'inherit' }}  />
 
```

however I refactored to 

```
<Input {...inputProps} _focus={{boxShadow: { md: isInputFocused ? 'outline' : 'inherit'}}} />

```

and that resolved the error for me 

also I'm on chakra-ui/react@1.8.7 if that helps

**Reply by ricardohpl** [reply]
Thanks this solution helped me!

**Comment by ghost** [comment]
From my sample above. Just need to change from _hover='none' to _hover={{}}

**Reply by ghost** [reply]
worked.
